tests/unit/protocols/output/ec2.json

```diff
@@ -1,612 +1,1847 @@
 [
```

Comment:
high-level question; do we know if the new files cover all the existing cases we're deleting?